MonoGame.Framework.Content.Pipeline/Serialization/Intermediate/CharSerializer.cs

```diff
@@ -17,17 +17,21 @@ class CharSerializer : ElementSerializer<char>
 
         protected internal override char Deserialize(string[] inputs, ref int index)
         {
-            var str = inputs[index++];
-            if (str.Length == 1)
-                return XmlConvert.ToChar(str);
+            if (inputs.Length > 0)
```

Comment:
The PR would be much cleaner if you would have simply added:
```
if (inputs.Length == 0)
  return new char();
```
Instead of wrapping the existing code in an `if` block.